docs/networks/mainnet.md

```diff
@@ -0,0 +1,104 @@
+| `INDEXER_AGENT_GAS_PRICE_MAX`               | `--gas-price-max`               | `50`                                                    |
+
+In order to avoid collecting or claiming query fees below a certain threshold
+(e.g. below the cost of the two transactions), the following configuration
+option can be used.
+
+| Environment Variable                         | CLI Argument                      | Value                                                                                     |
+| -------------------------------------------- | --------------------------------- | ----------------------------------------------------------------------------------------- |
+| `INDEXER_AGENT_REBATE_CLAIM_THRESHOLD`       | `--rebate-claim-threshold`        | Minimum rebate (in GRT) received for an allocation to claim (Default: 200)                |
+| `INDEXER_AGENT_REBATE_CLAIM_BATCH_THRESHOLD` | `--rebate-claim-batch-threshold`  | Minimum total rebates (in GRT) before a batched claim is processed (Default: 2000)        |
+| `INDEXER_AGENT_VOUCHER_EXPIRATION`           | `--voucher-expiration`            | Time (in seconds) to permanently delete vouchers with too few query fees  (Default: 2160) |            
+
+### Indexer Service
+
+| Environment Variable                          | CLI Argument                    | Value                                            |
+| --------------------------------------------- | ------------------------------- | ------------------------------------------------ |
+| `INDEXER_SERVICE_ETHEREUM`                    | `--ethereum`                    | An Ethereum mainnet node/provider                |
+| `INDEXER_SERVICE_ETHEREUM_NETWORK`            | `--ethereum-network`            | `mainnet`                                        |
+| `INDEXER_SERVICE_INDEXER_ADDRESS`             | `--indexer-address`             | Ethereum address of mainnet indexer              |
+| `INDEXER_SERVICE_MNEMONIC`                    | `--mnemonic`                    | Ethereum mnemonic for mainnet operator           |
+| `INDEXER_SERVICE_NETWORK_SUBGRAPH_DEPLOYMENT` | `--network-subgraph-deployment` | `QmV614UpBCpuusv5MsismmPYu4KqLtdeNMKpiNrX56kw6u` |
+| `INDEXER_SERVICE_NETWORK_SUBGRAPH_ENDPOINT`   | `--network-subgraph-endpoint`   | `https://gateway.thegraph.com/network`           |
+| `INDEXER_SERVICE_CLIENT_SIGNER_ADDRESS`       | `--client-signer-address`       | `0x982D10c56b8BBbD6e09048F5c5f01b43C65D5aE0`     |
+
+### Graph Node
+
+| Environment Variable | CLI Argument     | Value                               |
+| -------------------- | ---------------- | ----------------------------------- |
+| `ethereum`           | `--ethereum-rpc` | `mainnet:<ethereum-json-rpc-url>`   |
+| `ipfs`               | `--ipfs`         | `https://ipfs.network.thegraph.com` |
+
+## Feature support
+
+> This describes indexing & querying features which are experimental or not fully supported for indexing & query rewards and arbitration.
```

Comment:
Is there an explanation of the feature support matrix table somewhere we can link to? 
If not it'd be good to explain it fully here.  Some questions come up when glancing at it: 

i.e. 
- what is the `Aliases` column and how are values there used by which components?  
- what is the process for moving features through the matrix?

Reply:
I added a new `feature-support-matrix.md` to provide these details in one place (and a link to the Governance Proposal + GIP)